pyproject.toml

```diff
@@ -43,7 +43,7 @@ dependencies = [
     "pillow>=10.1.0",
     "numpy>=1.26.0",
     "xxhash>=3.5.0",
-    "gepa[dspy]==0.0.17",
+    "gepa[dspy]==0.0.18",
```

Comment:
@LakshyAAAgrawal does this need to be [dspy]? It seems the dspy optional dependency group does not have any packages.

Reply:
Yes, GEPA by default pulls in some dependencies, which may conflict with DSPy. Which is why I have created a dependency free version of GEPA for dspy under gepa[dspy]

Reply:
@LakshyAAAgrawal It seems gepa doesn't have any core dependencies though?
What package do you think is installed by default?
https://github.com/gepa-ai/gepa/blob/main/pyproject.toml#L20

Reply:
That is true, but the idea is that even if gepa introduces any default dependencies in the future, it should have a separate dependency list so that it never introduces any conflict for dspy.

Reply:
Iirc, default depencies are installed even if you add [dspy]. So it won't serve for the purpose I think.